app/services/stepfunctions/RegularContributionsClient.scala

```diff
@@ -35,15 +35,15 @@ object CreateRegularContributorRequest {
   implicit val decoder: Decoder[CreateRegularContributorRequest] = deriveDecoder
 }
 case class CreateRegularContributorRequest(
-  firstName: String,
-  lastName: String,
-  country: Country,
-  state: Option[String],
-  contribution: Contribution,
-  paymentFields: Either[StripePaymentToken, PayPalPaymentFields],
-  ophanIds: OphanIds,
-  referrerAcquisitionData: ReferrerAcquisitionData,
-  supportAbTests: Set[AbTest]
+    firstName: String,
```

Comment:
Are intentional all this amount of `tab` changes?

Reply:
This happened because of the scalariform upgrade.

There aren't that many changes and I think it makes them more consistent with the rest of the code base.